olm/eclipse-che-preview-kubernetes/Dockerfile

```diff
@@ -1,17 +1,25 @@
-FROM quay.io/operator-framework/upstream-registry-builder:v1.15.1 as builder
+#!/bin/bash
+#
+# Copyright (c) 2012-2018 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+# Contributors:
+#   Red Hat, Inc. - initial API and implementation
 
-COPY deploy manifests/eclipse-che-preview-kubernetes
-RUN ./bin/initializer -o ./bundles.db
+FROM quay.io/operator-framework/upstream-registry-builder:v1.15.3 as builder
 
-FROM openshift/origin-base
```

Comment:
I am a bit concerned of changing this image.
Do we have any issues with the current one?

Reply:
The most examples in appr-registry are using https://github.com/operator-framework/operator-registry/blob/master/appr-registry.Dockerfile#L20

Reply:
Thank you. It make sense then.